crypto-test/.js/src/test/scala/org/bitcoins/crypto/SigningTest.scala

```diff
@@ -0,0 +1,140 @@
+package org.bitcoins.crypto
+
+import org.scalatest.Assertion
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.must.Matchers
+import scodec.bits.ByteVector
+
+import scala.util.{Failure, Success, Try}
+
+class SigningTest extends AnyFlatSpec with Matchers {
+
+  it must "be able to sign and verify signatures" in {
+    val privkey = ECPrivateKey.freshPrivateKey
+    val pubkey = privkey.publicKey
+    val msg = BCryptoCryptoRuntime.randomBytes(32)
+    val sig = privkey.sign(msg)
+    assert(pubkey.verify(msg, sig))
+  }
+
+  it must "be able to sign and verify Schnorr signatures" in {
+    val privkey = ECPrivateKey.freshPrivateKey
+    val pubkey = privkey.publicKey
+    val msg = BCryptoCryptoRuntime.randomBytes(32)
+    val sig = privkey.schnorrSign(msg)
+    assert(pubkey.schnorrVerify(msg, sig))
+  }
+
+  val lines = Vector(
+    "index,secret key,public key,aux_rand,message,signature,verification result,comment",
```

Comment:
why not put this CSV file in a shared space (and not in the actual test code)?

Reply:
The code is the only shared space with javascript.

Reply:
Some more context here, javascript does not give us the ability to read files off the classpath (in `resources/`) -- which we do in the JVM with test files.